test/suites/perf/suite_test.go

```diff
@@ -55,7 +65,17 @@ var _ = BeforeEach(func() {
 	env.BeforeEach()
 	nodeClass = env.DefaultNodeClass()
 	nodePool = env.DefaultNodePool(nodeClass)
+	test.ReplaceRequirements(nodePool, v1beta1.NodeSelectorRequirementWithMinValues{
+		NodeSelectorRequirement: v1.NodeSelectorRequirement{
+			Key:      v1alpha1.InstanceSizeLabelKey,
+			Operator: v1.NodeSelectorOpLt,
+			Values:   []string{"32"},
+		},
+	})
```

Comment:
Added this so that we just have a lower instance size and get multiple nodes on simple scale ups.